glide.yaml

```diff
@@ -34,12 +35,13 @@ import:
   - proto
   - sortkeys
 - package: k8s.io/kubernetes
-  version: ~v1.6.0
+  version: 477efc3cbe6a7effca06bd1452fa356e2201e1ee
   subpackages:
   - pkg/client/leaderelection
   - pkg/client/leaderelection/resourcelock
   - pkg/client/clientset_generated/internalclientset/typed/core/internalversion
   - pkg/kubelet/types
 - package: github.com/golang/glog
-testImport:
```

Comment:
Yes, in the description I mentioned glide knows the mocks package is needed in source code. So the package is treated as a normal import. If you try to pin it, it will notice the import and test import are in conflict. Fix is to list the mock package as a regular import and pin it there. `testImport` is then no longer needed